lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -0,0 +1,87 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    # ![Definition demo](../../definition.gif)
+    #
+    # The [definition
+    # request](https://microsoft.github.io/language-server-protocol/specification#textDocument_definition) jumps to the
+    # definition of the symbol under the cursor.
+    #
+    # Currently supported targets:
+    # - Callbacks
+    #
+    # # Example
+    #
+    # ```ruby
+    # before_action :foo # <- Request go to definition on this symbol will take you to the method
+    # ```
```

Comment:
Let's add a note that this is currently only available if the method called method is defined in the same class?